content/blog/news/gsoc-2020-cloud-plugin.md

```diff
@@ -0,0 +1,172 @@
+under the `cuegui/config` directory to setup your GCP project name and the zone name that the API will use to access the resources from.
+
+6. At least one instance template setup on the cloud: I was using a RQD Blender docker image uploaded as an
+[Instance Template](https://cloud.google.com/compute/docs/instance-templates) with its appropriate VM type. 
+Few things to note here regarding Instance Templates:
+
+    * The docker image I used launched RQD using the container's startup command so I didn't have to explicitly specify any startup script.
+    
+    * If you are using a custom setup/running GCE instances without containers, make sure the startup script for the VM launches RQD when the VM is provisioned.
+    
+    * The `CUEBOT_HOSTNAME` environment variable must be set for RQD to point to Cuebot. In my deployment, this env variable was set to the internal IP address of Cuebot running on GCE.
+
+### Cloud plugin interface
+
+The Cloud plugin should be available to launch from the menu bar by navigating:
+**Views/Plugins -> CueCommander -> Cloud Plugin**
+
+1. The cloud plugin interface consists mainly of two components:
+    1. Main window which shows the list of cloud groups available with all registered providers.
+    2. A cloud group addition dialog that can be used to create a cloud group from the interface itself.
+
+### Cloud Plugin functionality
+
+1. To start create a cloud pool group, use the `Add Cloud Group` button to launch the
+dialog that will allow you to create a cloud group.
+
+![Add Cloud Group Dialog](/blog/images/AddCloudGroupDialog.png)
+
+2. If the API request is successful you should now see an entry in the Cloud Plugin's main interface.
+
+![After cloud group addition](/blog/images/MainInterfacePostAddition.png)
+
+3. The right-click content menu has two additional functionality:
+    1. Resizing (scaling up/down) the number of instances in a group.
+    2. Deleting the group altogether.
+
+![Cloud Plugin Context Menu](/blog/images/CloudPluginContextMenu.png)
+
+4. Once these cloud instances startup and RQD launches, you should see these nodes appear in the Monitor Hosts plugin, after which OpenCue will be able to send jobs to these hosts.
+
+## Further work and updates
+
+1. An Azure plugin that uses the same interface is currently a work in progress.
+A draft PR for the same can be found here: [Azure Plugin PR](https://github.com/AcademySoftwareFoundation/OpenCue/pull/771).
+
+2. [Local operation caching](https://github.com/AcademySoftwareFoundation/OpenCue/issues/773).
+
+3. [AWS integration](https://github.com/AcademySoftwareFoundation/OpenCue/issues/774) for the cloud plugin.
+
+4. Better integration with the MonitorHosts plugin: [Cloud group source for MonitorHosts plugin](https://github.com/AcademySoftwareFoundation/OpenCue/issues/775).
```

Comment:
To end this post I would restate the contact info -- reach out to opencue-user@ with any questions.